src/containers/BooksList.js

```diff
@@ -0,0 +1,20 @@
+import Book from '../components/Book';
+
+const BooksList = () => (
+  <div>
+    <h1>Books List</h1>
+    <table>
+      <tr>
+        <th>Book ID</th>
+        <th>Title</th>
+        <th>Category</th>
+      </tr>
+      <tbody>
+        <Book id={1} title="about memory" category="self improve" />
+        <Book id={2} title="dfdfdfd emory" category="ererererev improve" />
+      </tbody>
+    </table>
+  </div>
+);
+
+export default BooksList;
```

Comment:
- The `BookList` component should be connected to Redux store and fetches books from the state ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/ConnetToRedux.png))

- This component should also use mapStateToProps to fetch books from the store

- Lastly, you should pass a book object as a prop to each Book component. You do not have to break down the properties of the book. Kindly pass the book object.

These are features in Milestone 2, as seen in the project specification.
![save](https://user-images.githubusercontent.com/26856491/109535590-0a96e180-7abd-11eb-9ae2-67758434e2e0.png)